utils/util.js

```diff
@@ -22,19 +24,31 @@ exports.generateESIndexID = function(keys){
   return id;
 }
 
-exports.convertToEsDocument = function (doc) {
-  for (const key in doc) {
-    if (!doc.hasOwnProperty(key)) {
+/**
+ * Converts a DynamoDB payload into a document that can be inserted into ES.
+ * @param {Object} doc DynamoDB payload
+ * @returns {Object} Elastic document
+ */
+exports.convertToEsDocument = function (payload) {
+  const stringifiedNestedFields = [
+    'DATA_SCIENCE', 'DESIGN', 'DEVELOP', 'maxRating'
```

Comment:
@dhruvit-r is it possible to dynamically detect if the value is a json string, then parse it?

Reply:
@skyhit I've updated the code to make the check dynamic. Do take a look.